ibm/service/database/resource_ibm_database.go

```diff
@@ -2209,6 +2237,14 @@ func resourceIBMDatabaseInstanceUpdate(context context.Context, d *schema.Resour
 				change.Old = nil
 			}
 
+			newName, ok := change.New["name"]
+			isPgOrEdb := strings.Contains(service, "postgresql") || strings.Contains(service, "enterprisedb")
+			if ok && isPgOrEdb && newName.(string) == "repl" {
+				updateReplUser(change.New["password"].(string), instanceID, change.New["type"].(string), meta, d)
```

Comment:
if this method is necessary can it be a generic `changeUserPassword` method?

Reply:
Done :D